develop/components/pages/DropdownPage.tsx

```diff
@@ -11,6 +11,11 @@ const DropdownPage: React.FunctionComponent = () => {
     const [dropDownList4Selected, setDropdownList4Selected] = React.useState<DropdownItem>(null);
     const [dropDownList5Selected, setDropdownList5Selected] = React.useState<Array<DropdownItem>>([]);
     const [disabled, setDisabled] = React.useState<boolean>(false);
+    const [error, setError] = React.useState<string>(null);
```

Comment:
```suggestion
    const [error, setError] = React.useState<string>("");
```
To avoid showing null in the error field in some edge cases.